README.md

```diff
@@ -1,2 +1,52 @@
-# snapml-examples
-IBM Snap ML Examples
+# IBM Snap ML Examples
```

Comment:
Do we introduce somewhere the acronym Snap ML?

Reply:
added it